CMakeLists.txt

```diff
@@ -137,7 +137,7 @@ find_package(Qt6 COMPONENTS Core Concurrent Network REQUIRED)
 # POCO config
 find_package(Poco REQUIRED COMPONENTS Foundation Zip)
 
-function(determine_version OUTPUT_VARIABLE)
+function(determine_version OUTPUT_VARIABLE SEMANTIC_VERSION BUILD_NUMBER)
```

Comment:
`SEMANTIC_VERSION BUILD_NUMBER` is unnecessary now, right?

Reply:
yup, good catch